src/MongoDB.Driver.Core/Core/Compression/Snappy/Snappy64NativeMethods.cs

```diff
@@ -90,7 +90,7 @@ public override string GetLibraryRelativePath(SupportedPlatform currentPlatform)
                 switch (currentPlatform)
                 {
                     case SupportedPlatform.Windows:
-                        return "Core/Compression/Snappy/lib/win/snappy64.dll";
+                        return @"..\..\x64\native\windows\";
```

Comment:
Path is relative to the MongoDB.Driver.Core.dll's location inside the nuget package